jobs.groovy.j2

```diff
@@ -0,0 +1,433 @@
+{% macro folder(folder, display_name)                                       -%}
+{# creates a folder structure to group jobs
+   param folder:  '<clusterhq-flocker/branchname>'
+   param displayName: Name displayed in the jenkins view for this folder     #}
+println("creating {{folder}}...")
+folder("{{folder}}") { displayName("{{display_name}}") }
+{%- endmacro                                                                -%}
+{#
+
+#}
```

Comment:
it has a good bunch of comments in my view, the groovy code in it essentially the jenkins DSL as everything else has been abstracted in jinja2.
where do you think its lacking ?